everestrs/everestrs/Cargo.toml

```diff
@@ -4,7 +4,7 @@ version = "0.20.0"
 edition = "2021"
 
 [dependencies]
-argh = "0.1.10"
+clap = { version = "4.5.27", features = ["derive"] }
```

Comment:
argh does not support the renaming of args into camel case. So we either should use "some-thing" instead of "some_thing" or -  as done here change to clap